src/mono/mono.proj

```diff
@@ -81,4 +88,84 @@
 
   <Target Name="Test" />
 
+  <!-- Copy Mono runtime bits to the local .dotnet dir for local experiments (temp solution) -->
+  <Target Name="PatchLocalDotnet" DependsOnTargets="ValidateLocalDotnet">
+    <PropertyGroup>
+      <LocalDotnetRuntimeDir>$([System.IO.Directory]::GetDirectories("$(LocalDotnetDir)\shared\Microsoft.NETCore.App")[0])</LocalDotnetRuntimeDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <_MonoRuntimeArtifacts Include="$(BinDir)\*.*" />
+    </ItemGroup>
+    <Copy Condition="!Exists('$(LocalDotnetRuntimeDir)\System.Private.CoreLib.dll')"
```

Comment:
I guess you meant:

```suggestion
    <Copy Condition="!Exists('$(LocalDotnetRuntimeDir)\System.Private.CoreLib.dll-coreclr')"
```

Reply:
but I think a better way would be to remove this backing up of the original libcoreclr+SPC and instead in the RestoreLocalDotnet target copy them from `$(CoreCLRArtifactsPath)`

Reply:
there is a chance .dotnet won't work with libcoreclr built from master (e.g. some type was moved/removed). But I agree lets keep it cleaner